src/ontology/uberon-edit.obo

```diff
@@ -224760,147 +224760,134 @@ property_value: dcterms-date "2024-06-12T13:51:05Z" xsd:string
 [Term]
 id: UBERON:8910000
 name: stomatogastric nervous system
-def: "The part of the nervous system that controls the stomach, such as in crabs, lobsters, and flies" [FlyBase:FBrf0089570, PMID:12966498, PMID:27450880, DOI:10.1016/B0-12-370878-8/00177-4]
+def: "The part of the nervous system that controls the stomach, such as in crabs, lobsters, and flies" [DOI:10.1016/B0-12-370878-8/00177-4, FlyBase:FBrf0089570, PMID:12966498, PMID:27450880]
 is_a: UBERON:0011216 ! organ system subdivision
-relationship: in_taxon NCBITaxon:6656 ! Arthropoda
 relationship: dc-contributor https://orcid.org/0000-0002-1909-7004
+relationship: in_taxon NCBITaxon:6656 ! Arthropoda
 
 [Term]
 id: UBERON:8910001
 name: stomatogastric ganglion
-def: "The stomatogastric ganglion (STG) consists of about 30 neurons that form two central pattern generator circuits in crustaceans. STG neurons have large soma (~50-100 um) and have complex branches. The STG contains the motor neurons that innervate the striated muscles that move the gastric mill and pyloric regions of the stomach." [PMID: 17009928] 
-synonym: "STG" EXACT [PMID: 17009928]
+def: "The stomatogastric ganglion (STG) consists of about 30 neurons that form two central pattern generator circuits in crustaceans. STG neurons have large soma (~50-100 um) and have complex branches. The STG contains the motor neurons that innervate the striated muscles that move the gastric mill and pyloric regions of the stomach." [PMID:17009928]
+synonym: "STG" EXACT [PMID:17009928]
```

Comment:
This should have been flagged in the initial #3445 PR, but since we’re on fixing issues caused by that PR, I agree we might as well fix that one also.

Reply:
Thank you!